controls/srg_gpos/SRG-OS-000342-GPOS-00133.yml

```diff
@@ -11,4 +11,5 @@ controls:
             - rsyslog_encrypt_offload_actionsendstreamdrivermode
             - rsyslog_encrypt_offload_defaultnetstreamdriver
             - rsyslog_remote_loghost
+            - auditd_audispd_configure_sufficiently_large_partition
```

Comment:
This rule has been removed previously based on a discussion in https://github.com/ComplianceAsCode/content/pull/8924. What changed or why do we add it back?

Reply:
We might need a new rule for the audited version of this. The idea is valid, but the check is not.